midend/simplifyKey.h

```diff
@@ -121,10 +121,10 @@ class DoSimplifyKey : public Transform {
 };
 
 /**
- *  This pass considers any non-lvalues, except built-in method invocation (isValid),
- *  in table keys as complex expressions, and simplifies the keys.
- *
- * @param policy Use 'policy' to determine whether a key expression is too complex.
+ * This pass uses 'policy' to determine whether a key expression is too complex.
+ * Policies are defined in 'NonLeftValue' and 'NonMaskLeftValue', which consider
```

Comment:
This pass should not list all possible policies; we can't be updating this comment every time we write a new policy.
Each policy has its own comments. This pass also simplifies keys which are complex according to the policy.